src/backend/utils.ts

```diff
@@ -633,10 +633,15 @@ async function searchForExecutableOnPath(executable: string): Promise<string> {
   }
 }
 async function getSteamRuntime(
-  requestedType: 'scout' | 'soldier'
+  requestedType: 'scout' | 'sniper' | 'soldier'
```

Comment:
While you're touching this, might as well make this a little easier to work with:
```suggestion
  requestedType: SteamRuntime['type']
```